e2e/tests/block/k8s/integration/k8sBlockStorageIntegration_test.go

```diff
@@ -0,0 +1,159 @@
+package integration
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/rook/rook/e2e/framework/clients"
+	"github.com/rook/rook/e2e/framework/contracts"
+	"github.com/rook/rook/e2e/framework/enums"
+	"github.com/rook/rook/e2e/framework/manager"
+	"github.com/rook/rook/e2e/framework/objects"
+	"github.com/rook/rook/e2e/framework/utils"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"html/template"
+	"testing"
+)
+
+// Rook Block Storage integration test
+// Start MySql database that is using rook provisoned block storage.
+// Make sure database is functional
+var (
+	env      objects.EnvironmentManifest
+	platform enums.RookPlatformType
+)
+
+//One init function per package - initializes Rook infra and installs rook(if needed based on flags)
+func init() {
```

Comment:
init is per package, I am looking at ways to have a common init to do setup(install rook if needed) before each suite run. Since our tests are spread across multiple packages, I have one init per package.